app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
-        <div class="one-half">
-          <div class="inner-block">
-            <% if @prime_minister.present? && @prime_minister.image_url(:s465).present? %>
-              <%= image_tag @prime_minister.image_url(:s465), { alt: @prime_minister.name } %>
-            <% else %>
-              <%= image_tag 'how-gov-works/10_Downing_Street.jpg', alt: 'Number 10, Downing Street' %>
-            <% end %>
-          </div>
-        </div>
-      </div>
-
-      <div class="halves-width-section cabinet">
-        <div class="one-half">
-          <div class="inner-block">
-            <%= render "govuk_publishing_components/components/heading", {
-              text: "The Cabinet",
-              heading_level: 3,
-              margin_bottom: 5,
-            } %>
-
-            <p class="govuk-body">The Cabinet is made up of the senior members of government. Every week during Parliament, members of the Cabinet (Secretaries of State from all departments and some other ministers) meet to discuss the most important issues for the government.</p>
+  </div>
+</div>
 
-            <p class="govuk-body"><%= link_to("See who is in the Cabinet", "/government/ministers", class: "govuk-link") %></p>
-          </div>
-        </div>
-        <div class="one-half">
-          <div class="inner-block">
-            <%= image_tag 'how-gov-works/Cabinet_01.jpg', alt: 'The Cabinet' %>
-          </div>
-        </div>
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <div class="govuk-!-margin-bottom-9">
+        <%= render "govuk_publishing_components/components/contents_list", {
+          aria_label: "Pages in this guide",
+          contents: [
+            {
+              href: "#who-runs-government",
+              text: "Who runs government",
+            },
+            {
+              href: "#how-government-is-run",
+              text: "How government is run",
+            },
+            {
+              href: "#civil-service",
+              text: "Civil service",
+            },
+            {
+              href: "#get-involved",
+              text: "Get involved",
+            },
+            {
+              href: "#legislation",
+              text: "Legislation",
+            },
+            {
+              href: "#access-to-information",
+              text: "Access to information",
+            },
+            {
+              href: "#devolved-government",
+              text: "Devolved government",
+            },
+            {
+              href: "#local-government",
+              text: "Local government",
+            },
+            {
+              href: "#parliament",
+              text: "Parliament",
+            },
+            {
+              href: "#history-uk-government",
+              text: "History of government",
+            }
+          ]
+        } %>
       </div>
 
-      <div class="full-width-section ministers">
+      <section>
```

Comment:
I'm glad you used this. I'm not sure we have consistent, recommended agreement on `<section>` use. 

The [Design System layout](https://design-system.service.gov.uk/styles/layout/) doesn't mention it however it is [used here](https://github.com/alphagov/govuk-frontend/blob/4b3c6563a4cffa5b50763231b007b30b60623ea1/app/views/full-page-examples/campaign-page/index.njk) but in a different way.  I've raised a "?" about this [previously](https://github.com/alphagov/gds-way/issues/617) as I'm a bit unclear.

My stance at the moment is that we _should_ be using it [as it benefits some users](https://tink.uk/screen-reader-support-for-html5-sections/) but _how_ we use it I think we need to document a little.  

I would imagine it's OK to use in layout like this but the use above is slightly different and I'm not 100% it's supported.  Trying to find out.

   ```
   <section class="govuk-grid-row">
      <div class="govuk-grid-column-two-thirds">
        <h1 class="govuk-heading-xl">Two-thirds column</h1>
      </div>
    </section>

Reply:
Thanks, will have a look around to see if I can find anything helpful to add to the conversation too

Reply:
I'd tend towards divs until we have a proper strategy around when to use section, but it's probably not worth changing at this point unless it seems to cause any issues.